prov/util/src/util_attr.c

```diff
@@ -132,6 +132,41 @@ const char *ofi_core_name(const char *str, size_t *len)
 
 }
 
+int ofi_exclude_prov_name(char **prov_name, const char *util_prov_name)
```

Comment:
Please rename prov_name t prov_list or prov_name_list

Reply:
Thanks! I'll rename the parameter.